agent/internal/taskoutput/test_log.go

```diff
@@ -68,13 +68,16 @@ func newTestLogDirectoryHandler(dir string, output *taskoutput.TaskOutput, taskO
 		if err != nil {
 			return nil, errors.Wrap(err, "making test log sender")
 		}
-		return redactor.NewRedactingSender(evgSender, redactionOptions), nil
+		return redactor.NewRedactingSender(evgSender, redactionOpts), nil
 	}
 
 	return h
 }
 
 func (h *testLogDirectoryHandler) run(ctx context.Context) error {
+	ctx, span := tracer.Start(ctx, "test-log-ingestion")
```

Comment:
would we want to add more attributes to the span? For example, the `logFileCount` and the name of the directory